product-create-search/src/BusinessLogic/SendToKafka.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+
+namespace App\BusinessLogic;
+
+
+use Kafka\Producer;
+use Kafka\ProducerConfig;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class SendToKafka implements SendToKafkaI
+{
+    private $crudMysql;
+    private $status;
+    private $serializer;
+
+    public function __construct(CrudMysqLI $crudMysqli,  SerializerInterface $serializer)
+    {
+        $this->crudMysql = $crudMysqli;
+        $this->serializer = $serializer;
+    }
+
+    public function sendToKafka($status)
```

Comment:
can you count how many things this function is doing , first principle single responsibility